src/targets/gpu/CMakeLists.txt

```diff
@@ -252,6 +254,9 @@ endif()
 message(STATUS "clang-offload-bundler: ${MIGRAPHX_OFFLOADBUNDLER_BIN}")
 message(STATUS "extractkernel: ${MIGRAPHX_EXTRACT_KERNEL}")
 
+target_compile_definitions(migraphx_gpu PRIVATE "-DMIGRAPHX_MLIR_MIOPEN_SUPPORT")
```

Comment:
We should have an explicit cmake flag to enable MLIR: `set(MIGRAPHX_ENABLE_MLIR Off CACHE BOOL "")` and then only add those flags if it is set.